thrift/src/main/thrift/atoms/explainer.thrift

```diff
@@ -2,16 +2,59 @@ namespace * contentatom.explainer
 namespace java com.gu.contentatom.thrift.atom.explainer
 #@namespace scala com.gu.contentatom.thrift.atom.explainer
 
+include "../shared.thrift"
+
 enum DisplayType {
   FLAT = 0,
   EXPANDABLE = 1,
   CAROUSEL = 2
 }
 
 struct ExplainerAtom {
+  1: optional string label
   /* the unique ID will be stored in the `atom` data*/
   2: required string title
+  /* hopefully we can get rid of this one later: */
   3: required string body
```

Comment:
Why would we get rid of this field later? If this indeed is the intention, please mark it as optional so that we can remove it without breaking the model.